.circleci/config.yml

```diff
@@ -103,8 +116,8 @@ commands:
           when: always
           command: |
             mkdir -p ~/test-results/junit
-            find . -type f -regex ".*/target/surefire-reports/.*xml" -exec cp {} ~/test-results/junit/ \;
-            find . -type f -regex ".*/target/failsafe-reports/.*xml" -exec cp {} ~/test-results/junit/ \;
+            find . -type f -regex ".*/target/surefire-reports-[0-9]+/.*xml" -exec cp {} ~/test-results/junit/ \;
+            find . -type f -regex ".*/target/failsafe-reports-[0-9]+/.*xml" -exec cp {} ~/test-results/junit/ \;
```

Comment:
Would using
 $ find ... | xargs -r ....
be more performant here?

This is walking the full build tree twice to copy files over.  Could this be condensed into one statement?

Reply:
Well, you can't use xargs because the output of the find needs to be _before_ the target   path, and it appears support for extended regex and other things are spotty between OSes in  `find`, but you could do something like:

```
find . -type f -regex '.*/target/.*-reports.*\.xml' | grep -E '(surefire|failsafe)-reports' > /tmp/$$.filelist
rsync -v --files-from=/tmp/$$.filelist . ~/test-results/junit/
rm /tmp/$$.filelist
```

Reply:
Considering this step take under 20 seconds on average I think keeping it simple like this is acceptable.